src/cmd/vm.rs

```diff
@@ -176,69 +178,243 @@ fn enable_kvm() -> Result<()> {
+    /// you launch a local VM instance.
     #[argh(option)]
-    board: String,
+    board: Option<String>,
 
-    /// reuse the VM image. It is true by default. If you want to disable it,
-    /// use `--reuse-disk-image false`.
+    /// for betty.sh. Reuse the VM image. It is true by default. If you want to
+    /// disable it, use `--reuse-disk-image false`.
     #[argh(option, default = "true")]
     reuse_disk_image: bool,
 
-    /// start betty with rootfs verification. It is false by default.
+    /// for betty.sh. Start betty with rootfs verification. It is false by
+    /// default.
     #[argh(switch)]
     rootfs_verify: bool,
 
-    /// the ChromeOS version to use (e.g. R72-11268.0.0). Alternatively,
-    /// postsubmit builds since R96-14175.0.0-53101 can also be specified. It is
-    /// the latest version by default.
+    /// for betty.sh. The ChromeOS version to use (e.g. R72-11268.0.0).
+    /// Alternatively, postsubmit builds since R96-14175.0.0-53101 can also
+    /// be specified. It is the latest version by default.
     #[argh(option)]
     version: Option<String>,
 
-    /// path to betty VM image to start. It has priority over --board and
-    /// --version (they will be ignored)
+    /// for betty.sh. Path to betty VM image to start. It has priority over
+    /// --board and --version (they will be ignored)
     #[argh(option)]
     vm_image: Option<String>,
 
-    /// extra arguments to pass to betty.sh. You can pass other options like
-    /// --extra-args "options".
+    /// for acloudw. Launch a cloud based VM instance. It is false by default.  
+    #[argh(switch)]
+    acloud: bool,
+
+    /// for acloudw. The Android branch. It is required if --acloud is
+    /// specified.
+    #[argh(option)]
+    branch: Option<String>,
+
+    /// for acloudw. The Android Build ID. It is required if --acloud is
+    /// specified.
+    #[argh(option)]
+    build_id: Option<String>,
+
+    /// for acloudw. Select ARC-container, not ARCVM. It is false by default.
+    #[argh(switch)]
+    container: bool,
+
+    /// extra arguments to pass to betty.sh or acloudw. You can pass other
+    /// options like --extra-args "options".
     #[argh(option)]
     extra_args: Option<String>,
 }
 
 fn run_start(args: &ArgsStart) -> Result<()> {
+    if args.acloud {
+        run_acloudw(args)?;
+    } else {
+        run_betty_start(args)?;
+
+        println!("To connect the betty instance, run `lium dut shell --dut localhost:9222`.");
+        println!("To push an Android build a betty VM, run `lium arc flash`.");
+    }
+
+    Ok(())
+}
+
+fn run_acloudw(args: &ArgsStart) -> Result<()> {
+    let branch = args
+        .branch
+        .clone()
+        .ok_or(anyhow!("--branch option is required when using acloudw"))?;
+    let git_branch = format!("git_{branch}");
+
+    // b/314731302 is the internal bug tracker about getting the latest Android
+    // build ID
```

Comment:
Sorry I think you should only say relevent info so

b/314731302 use the android api to get this progromatically if not specified

Reply:
Fixed at https://github.com/google/lium/pull/201/commits/5d268a07c794a8ab31f29beef4b3e3fda89a6c4c !